salt-blackout-formula/salt-blackout-formula.spec

```diff
@@ -0,0 +1,56 @@
+#
+# spec file for package salt-blackout-formula
+#
+# Copyright (c) 2019 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+
+%define fname salt-blackout
+%define fdir  %{_datadir}/susemanager/formulas
```

Comment:
Maybe we can start using the new `/usr/share/salt-formulas` directory as requested in https://github.com/SUSE/spacewalk/issues/10230